### PR TITLE
findfeaturessegment fixes

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

This fixes a bug where the app tries to merge cnets that failed to be created. 

## Related Issue

https://github.com/DOI-USGS/ISIS3/issues/5416

## How Has This Been Validated?

Tested locally with the issue authors data and successfully created the control network. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added my user impacting change to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document. 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Lynn was able to test it locally (python script W since I just needed to send her the new python script) and seems to be working for her. 

FFSeg_Test

File: isis/python_bindings/apps/findFeaturesSegment/findFeaturesSegment.py [hunk 1]
```diff
@@ -311,7 +313,6 @@ def findFeaturesSegment(ui):
     else: 
         nthreads = int(multiprocessing.cpu_count())
 
-
     pool = ThreadPool(ceil(nthreads/len(img_list)))
```
Comment (AustinSanders) [inline isis/python_bindings/apps/findFeaturesSegment/findFeaturesSegment.py:316]: Not part of the PR, but I just saw this line.  Is this correct..?  If the user passes in nthreads=8 and there are 8 images, then won't this end up with a threadpool of 1?

**Review by AustinSanders — APPROVED** [review]
Changes look good either way, but one question on existing code

File: isis/python_bindings/apps/findFeaturesSegment/findFeaturesSegment.py (hunk 1 above)
Comment (Kelvinrr) [reply isis/python_bindings/apps/findFeaturesSegment/findFeaturesSegment.py:316]: I feel like there was a reason for that but in hindsight it doesn't make sense 🤔 I'll look into it some more.
